pkg/controller/nodelifecycle/metrics.go

```diff
@@ -60,11 +61,21 @@ var (
 		[]string{"zone"},
 	)
 	evictionsNumber = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:         nodeControllerSubsystem,
+			Name:              evictionsNumberKey,
+			Help:              "Number of Node evictions that happened since current instance of NodeController started, This metric is replaced by node_collector_evictions_total.",
+			DeprecatedVersion: "1.23.0",
```

Comment:
This should probably be bumped to 1.24 since it missed code freeze.

Reply:
yes, i agree